.env.example

```diff
@@ -11,12 +11,16 @@ VUE_APP_BCONLINE_URL="https://d1.bconline.gov.bc.ca/"
 VUE_APP_BUSINESS_EDIT_URL="https://dev.edit.business.bcregistry.gov.bc.ca"
 VUE_APP_BUSINESS_CREATE_URL="https://dev.create.business.bcregistry.gov.bc.ca"
 VUE_APP_BUSINESS_FILING_URL="https://dev.business.bcregistry.gov.bc.ca/"
+VUE_APP_DOCUMENTS_UI_URL="https://dev.documents.bcregistry.gov.bc.ca/"
 
 #vaults API
 VUE_APP_AUTH_API_URL="https://auth-api-dev-142173140222.northamerica-northeast1.run.app"
 VUE_APP_AUTH_API_VERSION="/api/v1"
 VUE_APP_PAY_API_URL="https://pay-api-dev.apps.silver.devops.gov.bc.ca"
 VUE_APP_PAY_API_VERSION="/api/v1"
+VUE_APP_DOC_API_URL="https://bcregistry-dev.apigee.net/doc"
+VUE_APP_DOC_API_VERSION="/api/v1"
+VUE_APP_DOC_API_KEY=""
```

Comment:
Add these to vaults.env file, too.

And ask Patrick to configure the Dev API key in GCP.

Reply:
Absolutely, will do!